Update gems to make it work again

picandocodigo opened the pull request:
Not sure if the intention is to keep maintaining this gem, but I wanted to install it recently and found it didn't work with Ruby 2.4. I managed to fix it for me by updating the gems and replacing html-pipeline's SyntaxHighlightFilter with RougeFilter.

picandocodigo committed:
Update gems to make it work again
Replaced html-pipeline's SyntaxHighlightFilter with RougeFilter

jacksenechal commented:
This looks good to me. @adamlogic do you want to merge this and publish the updated gem?

picandocodigo committed:
Update gems, spec

picandocodigo commented:
@jacksenechal thanks for looking into this after so long. I updated some gems to a newer version. Will this still be maintained?

adamlogic commented:
Thanks for the work here. I haven't used this gem in years, but I'm happy to hand the keys over if one of you wants to maintain it.

picandocodigo commented:
Hi @adamlogic :wave: 
I still use the gem every now and then, so I'd be happy to maintain it!

adamlogic commented:
@picandocodigo Awesome! I just gave you write access to the repo. Need your email to add you as an owner for the gem. I don't see it listed in your GH profile, so you can email it to me if you want (my email address is listed in my GH profile).

picandocodigo committed:
Update on dependencies and preparing updated release

- Specify dependency versions to match :test in https://github.com/jch/html-pipeline/blob/master/Gemfile
- Update emoji assets url
- Add byebug for development

picandocodigo committed:
Update spec

picandocodigo committed:
Address some rubocop offenses

picandocodigo committed:
Adds Changelog

picandocodigo committed:
Final details before new release 0.2.0
